conda/recipes/rapids_cuda_patched_dependencies/build.sh

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+# Script assumes the script is executed from the root of the repo directory
+
+REPODIR=$(cd $(dirname $0); pwd)
+
+SOURCE_DIR=${REPODIR}/conda/recipes/rapids_cuda_patched_dependencies
+BUILD_DIR=${BUILD_DIR:=${REPODIR}/build}
+
+INSTALL_PREFIX=${INSTALL_PREFIX:=${PREFIX:=${CONDA_PREFIX:=$BUILD_DIR/install}}}
+export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
+
+for bd in ${BUILD_DIRS}; do
+  if [ -d "${bd}" ]; then
+      find "${bd}" -mindepth 1 -delete
+      rmdir "${bd}" || true
+  fi
+done
+
+cmake -S "${SOURCE_DIR}" -B "${BUILD_DIR}" \
+    -DCMAKE_INSTALL_LIBDIR="lib" \
+    -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
```

Comment:
I might be missing something. We need a "special" prefix to prevent these patched dependencies from being found directly in `include/`, right? Where is that defined?

Reply:
That logic is defined in the `rapids_cpm_thrust` and `rapids_cpm_libcudacxx` where we modify the `CMAKE_INSTALL_INCLUDEDIR` ( install into a nested `rapids` dir ).